.github/workflows/continuous-integration.yml

```
@@ -205,16 +205,26 @@ jobs:
         id: get-broken-link-info
         run: node ./script/github-actions/check-broken-links.js ${{ matrix.buildtype }}
 
+      - name: Generate build details
+        run: |
+          cat > build/${{ matrix.buildtype }}/BUILD.txt << EOF
+          BUILDTYPE=${{ matrix.buildtype }}
+          BRANCH_NAME=$(echo "${GITHUB_REF#refs/heads/}")
+          RUN_ID=${{ github.run_id }}
+          REF=${{ github.sha }}
+          BUILDTIME=$(date +%s)
+          EOF
+
       - name: Compress ${{ matrix.buildtype }} build
         if: ${{ github.ref == 'refs/heads/master' && success() }}
-        run: tar -zcvf ${{ matrix.buildType }}.tar.gz build
+        run: tar -jcvf ${{ matrix.buildType }}.tar.bz2 build/${{ matrix.buildType }}
 
       - name: Upload build
         uses: actions/upload-artifact@v2
         if: ${{ github.ref == 'refs/heads/master' && success() }}
         with:
           name: ${{ matrix.buildtype }}
```

This should probably have the suffix. When I was testing in `vets-website`, I found it was nice to have it automatically have the right extension when I downloaded the artifact. I have that change coming up over there too.

```suggestion
          name: ${{ matrix.buildtype }}.tar.bz2
```